src/renderer/utils/InstallerConfiguration.ts

```diff
@@ -152,6 +153,28 @@ export interface AddonDependency {
     modalText?: string,
 }
 
+/**
+ * fields from the addon's manifest.json
+ */
+export interface AddonIncompatibleAddon {
+    /**
+     * Fields from the addon's manifest.json
+     * This need to be configured identically to the addon's manifest.json
+     * Leaving a field empty ignores it for the search.
+     */
+    title?: string,
+    creator?: string,
+    package_version?: string,
```

Comment:
**style:** config properties use camelCase, so `packageVersion`

Reply:
ok will do - has been done on purpose as the manifest uses package_version - but either way is good for me